### DISCUSSION TITLE
v0.1.0-alpha.10 (with multi-arch support!)

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#80] [Releases]
## What's Changed
* When converting bits its to the power of 1000 not 1024 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/70
* Results time zones, datetime formats and readme improvements by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/71
* Setup Dependabot for automatic depdendency updates by @adamus1red in https://github.com/alexjustesen/speedtest-tracker/pull/68
* Dependency updates by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/79
* Add basic multi-arch build support by @adamus1red in https://github.com/alexjustesen/speedtest-tracker/pull/67

## New Contributors
* @adamus1red made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/68

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.1.0-alpha.9...v0.1.0-alpha.10

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.1.0-alpha.10'>v0.1.0-alpha.10 (with multi-arch support!)</a>.</em>

**Comment by alexdelprete** [comment]
I love your "soon(tm)". It really means soon...:D

Thanks @adamus1red for your work on this.

**Comment by JourneyOver** [comment]
@alexjustesen Timezones are still not actually being used from the looks of things as things are still being reported in UTC time even though timezone is set through the settings page..

Edit: I opened up an issue for this https://github.com/alexjustesen/speedtest-tracker/issues/84
